java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/LoggerPlugin.java

```diff
@@ -8,7 +8,7 @@
  */
 public class LoggerPlugin implements Plugin {
     /**
-     * Constructs a logger plugin. This constructor installs a custom logger in the communicator. The communicator takes
+     * Constructs a logger plugin and installs it in the provided communicator. The communicator takes
```

Comment:
"it" sounds like we are installing the "logger plugin" instead of the "logger".

Reply:
You're correct. Changed it to:
`Constructs a logger plugin and installs a custom logger in the provided communicator`